src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamespaceSymbol.vb

```diff
@@ -71,9 +74,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         End Function
 
         Public NotOverridable Overloads Overrides Function GetMembers() As ImmutableArray(Of Symbol)
-            EnsureAllMembersLoaded()
+            If _lazyFlattenedNamespacesAndTypes.IsDefault Then
+                EnsureAllMembersLoaded()
+                _lazyFlattenedNamespacesAndTypes = m_lazyMembers.Flatten()
```

Comment:
Strictly speaking, struct assignments are not guaranteed to be atomic. We should use ```ImmutableInterlocked.InterlockedExchange``` as we do in C# code.

Reply:
I went ahead and changed this. Reasoning I didn't originally:

1) Was following what the other ImmutableArray item did in the file (_lazyFlattenedTypes)
2) The ImmtableArray struct is only a single ptr, and thus shouldn't suffer from a tearing threat. But, I agree, better to not depend on that, and just follow what C# does.